src/interpolators.js

```diff
@@ -27,7 +29,48 @@ export function isTemplateVariable(str) {
  * Interpolates a string, replacing template variables with values
  * provided in the scope.
  *
- * Besides replacing variables with
+ * @param  {String}    str    A string to be interpolated
+ * @param  {Object}    scope  An object with variable names and their
+ *                            replacements
+ * @return {String}           An interpolated string
+ */
+export function interpolateString(str, scope = {}) {
+  if (!str) {
+    return str
+  }
+
+  // Split string into array with regular text and variables split
+  // into separate segments, like ['This is a ', '{{ thing }}', '!']
+  const parts = str.split(new RegExp(variableRegex)).filter(x => x)
+
+  // If the only thing we have is a single regular string, just return it as is
+  if (parts.length === 1 && isTemplateVariable(parts[0]) === false) {
+    return str
+  }
+
+  return parts
+    .map(part => {
+      if (isTemplateVariable(part) === false) {
+        return part
+      }
+
+      const variableName = part.replace(/^\{\{\s/, '').replace(/\s\}\}$/, '')
```

Comment:
I think we use this regular expression (`/^\{\{\s/, '').replace(/\s\}\}$/`) in two places now - it may be worth while creating a variable outside of this function to ensure we use the same logic. Unless we want them to be distinctly different.

Reply:
They are slightly different what they think they are fetching; one fetches a simple variable name, whilst the other fetches a possible `key:contents` sorta thing. For two cases, it's no biggie.